cmd/logs.go

```diff
@@ -140,16 +185,61 @@ func logPod(toMatch string, toTarget string, container string) {
 		Client, err = clientToTarget(toTarget)
 		checkError(err)
 	}
+	if !flags.elasticsearch {
+		showLogsFromKubectl(namespace, toMatch, container)
+	} else {
+		showLogsFromElasticsearch(namespace, toMatch, container)
+	}
+}
+
+func showLogsFromKubectl(namespace, toMatch, container string) {
+	if container != "" {
+		container = " -c " + container
+	}
 	pods, err := Client.CoreV1().Pods(namespace).List(metav1.ListOptions{})
 	checkError(err)
 	for _, pod := range pods.Items {
 		if strings.Contains(pod.Name, toMatch) {
-			err := ExecCmd(nil, "kubectl logs --tail="+numberOfLines+" "+pod.Name+container+" -n "+namespace, false, "KUBECONFIG="+KUBECONFIG)
+			err := ExecCmd(nil, buildKubectlCommand(namespace, pod.Name, container), false, "KUBECONFIG="+KUBECONFIG)
 			checkError(err)
 		}
 	}
 }
 
+func showLogsFromElasticsearch(namespace, toMatch, container string) {
+	commands := buildElasticsearchCommands(namespace, toMatch, container)
+	for _, command := range commands {
+		output, err := ExecCmdReturnOutput("bash", "-c", fmt.Sprintf("export KUBECONFIG=%s; %s", KUBECONFIG, command))
```

Comment:
Isn't possible to pass the path to kubeconfig with the `--kubeconfig` param of kubectl?